### DISCUSSION TITLE
Tooltip stays open

### DISCUSSION THREAD

**Discussion by davidbarman** [mantinedev/mantine#2184] [Q&A]
When using Tooltip wrapping a Button component.
Tooltip opens as desired on hover of the button.  During onClick of the Button, the Button is disabled.
In this condition, the Tooltip remains open and never closes unless the Button is re-enabled and then the Button is re-hovered and on mouseout the tooltip closes.
Is my implementation wrong? Or is this an issue with Tooltip?

Code is as follows:
` <Tooltip
                label="New Customer"
                openDelay={400}
                closeDelay={400}
                withArrow
                arrowSize={5}
                color="blue"
              >
                <Button
                  disabled={formMode != FORM_DISABLED_MODE}
                  onClick={() => {
                    setFormDisabled(false);
                    setFormMode(FORM_ADD_MODE);
                  }}
                >
                  <FaUserPlus style={{ color: "white" }} size={25} />
                </Button>
              </Tooltip>`

**Comment by rtivital** [comment]
This is bug in react with mouseleave event in disabled buttons, you need to use controlled tooltip with pointer events instead – https://codesandbox.io/s/nostalgic-hertz-b51vs2?file=/src/App.tsx

**Reply by davidbarman** [reply]
That worked.  Thanks.
I had two buttons in the component so I was able to rename the open, closed when destructuring the object.  Otherwise, worked just like you showed.
Thank you.

**Comment by zhulien-ivanov** [comment]
Had the same issue. You can find a related discussion with a workaround in discord's channel here: https://discord.com/channels/854810300876062770/1008445746246320210.

**Comment by rtivital** [comment]
Explanation from floating ui maintainers – https://github.com/floating-ui/floating-ui/discussions/1865
